regolith/builders/recentcollabsbuilder.py

```diff
@@ -2,6 +2,9 @@
 import os
 import datetime as dt
 from copy import copy
+from typing import List, Any, Tuple
+import pandas as pd
```

Comment:
can we remove pandas?  I am not convinced it is adding sufficient value here to (a) change the requirements of regolith and (b) introduce a new datastructure (panda's dataframe).  If you can convince me otherwise we can maybe justify it, but right now I don't see it.

Reply:
@sbillinge There are three options to parse and save the data in .csv file: (1) panadas; (2) built-in csv module; (3) jinja2.

(1) Pro: just one line of code (pd.DataFrame.to_csv), quick; Con: need install pandas module; may be overkill

Code:

pd.DataFrame([...]).to_csv('...')

(2) Pro: it is a build in model in python, quick; Con: four lines of code

Code:

with open('xxx.csv', "r") as f:
         w = csv.writer(f)
         w.write([...])
         for row in rows:
               w.write(row)

(3) Pro: do not need to worry about the build of directroy; Con: need templates; need to reformat the data into a list of string with "," delimeters.

Code:

rows = [','.join(tup) for tup in tups]
self.render('xx.csv', 'xx.csv', rows)

Template:

{% for row in rows %}
{{row}}

Pandas may be not necessary here. However, I am going to introduce the pandas in the requirements in regolith for my "beamplanbuilder" module bceaseuse it really needs that to generate the table in the latex form. I think it is ok to use pandas here to make the code simple.

Reply:
I forgot to mention that the Jinja2 is not capable to do this task.

For example, if there is "," in a human name, jinja2 cannot identify that it is a data instead of a delimeter and it will break the .csv format.

So our options are (1) pandas Or (2) csv